src/table/expandable-rows/expand-toggle-button.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useRef } from 'react';
+import { useSingleTabStopNavigation } from '../../internal/context/single-tab-stop-navigation-context';
+import styles from './styles.css.js';
+import InternalIcon from '../../icon/internal';
+import { ExpandableItemProps } from './expandable-rows-utils';
+import clsx from 'clsx';
+
+export function ExpandToggle({ isExpandable, ...props }: ExpandableItemProps) {
+  return isExpandable ? <ExpandToggleButton {...props} /> : <ExpandTogglePlaceholder />;
```

Comment:
Actually why do we render the ExpandTogglePlaceholder in case the row is not expandable, instead of just nothing?

Reply:
Great question. It was previously needed to keep the consistent offsets, but now that is actually handled with CSS of the body-cell. Will update.